raml-validate.js

```diff
@@ -285,19 +335,24 @@ module.exports = function () {
    * @param  {Object}   schema
    * @return {Function}
    */
-  function validate (schema) {
+  function validate (schema, RAMLVersion) {
     if (!schema) {
       return function () {
-        return { valid: true, errors: [] };
-      };
+        return { valid: true, errors: [] }
+      }
     }
 
-    var validations = {};
+    var validations = {}
+    RAMLVersion = RAMLVersion || 'RAML10'
 
     // Convert all parameters into validation functions.
     Object.keys(schema).forEach(function (param) {
```

Comment:
Minor issue: instead of checking RAMLVersion on each loop step, we could do something like:
```
var handler = RAMLVersion === 'RAML10' ? validate.ruleRAML10 : validate.rule
```
and then inside loop
```
validations[param] = handle(schema[param])
```